assets/src/scss/new-identity/style.scss

```diff
@@ -123,9 +123,12 @@ table.spreadsheet-table.is-color-gp-green {
 
 a:where(.standalone-link) {
   width: fit-content;
-  border-bottom: 2px solid var(--gp-green-500);
   font-weight: 700 !important;
-  transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
+}
+
+.standalone-link, p > a {
```

Comment:
Looking at the example page you created, one fix will be to add `.external-links` to the selector, another we can use the `post-details > a` selector, assuming all body copy links are present only within this context

Reply:
Adding `.external-links` is definitely not the way to go, in my example the links are external but it could not be the case. If we do `.post-details > a` the ones in links won't be included, since they won't be direct children of the `.post-details` element. However we could maybe use `.post-details` 🤔 Something like `.post-details li > a`, however this will only work in Posts because other post types use different classnames (I think for Pages it's `page-content`?)

Reply:
Maybe we should also use that for `p > a` actually, to make sure only the content is targeted and not other elements (footer, etc)